src/coremods/core_channel/cmd_invite.cpp

```diff
@@ -56,9 +56,14 @@ CmdResult CommandInvite::Handle (const std::vector<std::string>& parameters, Use
 				timeout = ConvToInt(parameters[3]);
 		}
 
-		if ((!c) || (!u) || (u->registered != REG_ALL))
+		if (!c)
 		{
-			user->WriteNumeric(Numerics::NoSuchNick(c ? parameters[0] : parameters[1]));
+			user->WriteNumeric(Numerics::NoSuchChannel(parameters[1]));
```

Comment:
This will always result in an error as a target cannot be both a user and a channel.

Reply:
The code that this is replacing is: ```if ((!c) || (!u) || (u->registered != REG_ALL))```

Git diffs isn't showing https://github.com/B00mX0r/inspircd/blob/1e6d366930fde3c21f0ebf3485983905ba62491a/src/coremods/core_channel/cmd_invite.cpp#L44,#L49, which is the relevant part that shows how both should be defined.